iron-fit-behavior.html

```diff
@@ -253,11 +253,13 @@
       if (typeof this._isRTL === 'undefined') {
         this._isRTL = window.getComputedStyle(this).direction == 'rtl';
       }
-          
       this.positionTarget = this.positionTarget || this._defaultPositionTarget;
       if (this.autoFitOnAttach) {
-        if (window.getComputedStyle(this).display === 'none') {
-          setTimeout(function() {
+        // If invisible or in a shadow root that's still not connected, wait a tick.
+        if (window.getComputedStyle(this).display === 'none' ||
+          document.body.contains(this) === false) {
```

Comment:
`attached` only called in gets called as a result of [`connectedCallback`](https://github.com/Polymer/polymer/blob/282c3afc1f22d75146b9e2638f62a4427bb60e46/lib/legacy/legacy-element-mixin.html#L70) (I don't think there are other places?) and `connectedCallback` reactions are only enqueued when an element's shadow-including root changes from a non-document to a document. So, running that snippet on its own won't cause `ce`'s`attached` to run. However, if `attached` *is* getting called as a result of that snippet, then this is likely a bug in the custom elements polyfill.